144 clean search page text

DamiOgboja opened the pull request:
# Proposed Changes

* Revamped Search page text
* Added pagination to search results

Fixes # .

DamiOgboja committed:
search page revamp

DamiOgboja committed:
searchbar added pagination +formatting

DamiOgboja committed:
added note, + formatting

this file is not used for anything right now since we do live search results but im keeping it in case i want to revamp later.

DamiOgboja committed:
formatting

DamiOgboja committed:
removed unused variables

mechamancy reviewed (changes requested):
Functionally, works great! Search page is now updated and more descriptive, and the pagination is really nice.
My only comment really is I personally would leave some spacing in between the search bar and the bottom of the page; on smaller screens, it gets a bit scrunched up. After that, I think it's good to merge.

DamiOgboja committed:
moved searchbar

by popular demand we've decided to move the search bar up Up UP! Now located right beneath the header, try out our new and improved SEARCH PAGE, no more of that needless scrolling and corny login and user benefits text, you know what we offer don't ya? NOW GET IN THERE!

okay now in all seriousness, search bar moved up, and text trimmed to clean it up and prevent needless scrolling

mechamancy reviewed (approved):
Having seen the changes both in screenshots and localhost, everything looks good and seems to work fine. Ready to merge
